.eslintrc.js

```diff
@@ -156,7 +160,7 @@ module.exports = {
         '@osd/eslint/require-license-header': [
           'error',
           {
-            licenses: [OSD_HEADER, OSS_HEADER],
+            licenses: [OSD_HEADER, OSD_MODIFIED_HEADER],
```

Comment:
You needn't modify the existing headers, just add a new variable say 

``` 
const OSS_HEADER_COMBINED = `
/*
 * SPDX-License-Identifier: Apache-2.0
 *
 * The OpenSearch Contributors require contributions made to
 * this file be licensed under the Apache-2.0 license or a 
 * compatible open source license.
 * 
 * Modifications Copyright OpenSearch Contributors. See
 * GitHub history for details.
 */
`;
```

and add that to the allowed licenses list as 

```
licenses: [OSD_HEADER, OSS_HEADER, OSS_HEADER_COMBINED]
```

This ensures that the combined header is also accepted without breaking compatibility for the older header that was split.

Reply:
My goal is to prevent incorrect headers to remain with new commits. I will update this PR and fix all the headers in all the files.